index.html

```diff
@@ -0,0 +1,134 @@
+<!doctype html>
```

Comment:
Great work on opening a PR and adding reviewers!

However, the point of a PR is generally to display changes. Often a PR will add many new files, but in this case files like `index.html` should only show changes, and files like `webpack.common.js` should already be in the `main` branch and not reflect in the PR.

Opening a PR where the entire project is created makes it difficult to see what is from the template and what is your code.

Reply:
Thanks Kyle, I''ll take note of that and I apologise for the way the changes are shown. I hadn't realised that changing the file directory would cause github to stop tracking changes I've made. I changed the directory because I wanted to deploy on github, and for that to be done, the index.html file has to be in the root folder. 

I believe the commits before my 'fixed-file-directory' commit, show changes which you can properly review and I apologise again for the inconvenience.

Reply:
No inconvenience at all! It just means that it's difficult for someone to give feedback.

Nice one on deploying to GitHub!! That's very helpful for someone to view the result and it's similar to what you'll be doing in the workplace one day.